tools/clang/tools/dxa/dxa.cpp

```diff
@@ -190,7 +189,7 @@ bool DxaContext::ExtractFile(const char *pName) {
 
   CA2W WideName(pName, CP_UTF8);
   for (UINT32 i = 0; i < uNumSources; i++) {
-    CComBSTR name;
+    BSTR name;
```

Comment:
Wouldn't this change cause name to leak?

Reply:
Fixed.